src/index.js

```diff
@@ -1,7 +1,9 @@
-export { ActionBar } from './ActionBar/ActionBar';
-export { ActionBarBack } from './ActionBar/ActionBar';
-export { ActionBarHeader } from './ActionBar/ActionBar';
-export { ActionBarActions } from './ActionBar/ActionBar';
+export {
+    ActionBar,
+    ActionBarBack,
+    ActionBarHeader,
+    ActionBarActions
+} from './ActionBar';
```

Comment:
These are being imported from the auto generated `index` file in the component directory. Let me know if I should import directly from the component files.